README.rst

```diff
@@ -76,18 +76,31 @@ than one CPU core, refer to our `parallel backend guide`_.
 
 **Note for Windows users**
 
-The pip installer for Neuron does not yet work for Windows. In this case, it is better to
-install ``hnn_core`` without the dependencies::
+The pip installer for Neuron does not yet work for Windows. In this case we recommend
+installing `WSL <https://docs.microsoft.com/en-us/windows/wsl/install>`_ on your
+local machine, and install ``hnn-core`` using the same steps as above.
 
-   $ pip install hnn_core --no-deps
-
-and then install the dependencies separately::
+Otherewise, to use ``hnn-core`` natively on windows, it is necessary to first install the
+dependencies separately::
 
    $ pip install scipy numpy matplotlib
 
-and install Neuron using the traditional package installer available here
+install Neuron using the traditional package installer available here
 `https://neuron.yale.edu/neuron/ <https://neuron.yale.edu/neuron/>`_.
 
+manually install ``hnn-core`` by cloning the latest stable release from github::
+
+   $ git clone https://github.com/jonescompneurolab/hnn-core --depth 1
+   $ cd hnn-core
+   $ git fetch --tags
+   $ latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
+   $ git checkout $latestTag
+
+and finally pip install the local copy, as well as build the Neuron .mod files::
+   
+   $ pip install -e .
+   $ python setup.py build_mod
```

Comment:
I don't think we should be encouraging regular users to do editable installs. You'll start to see users who messed up their local directory with git and then you have to teach them git :) The right fix is to update the MANIFEST.in and package_data directory correctly. The questions to answer are:

1. If you install Neuron on Windows, can it be installed within an Anaconda environment? See [here](https://web.archive.org/web/20201126015939/https://www.neuron.yale.edu/ftp/neuron/nrn_mswin_install.pdf)
2. If the mod files are built on Windows, what's the best way to copy the dll file to the Anaconda directory.

Step 2. is what doesn't get run in an editable install. We should fix it instead of suggesting users to do an editable install.